frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewSidePanel/FeatureOverviewSidePanelEnvironmentSwitches/FeatureOverviewSidePanelEnvironmentSwitches.tsx

```diff
@@ -89,6 +102,34 @@ export const FeatureOverviewSidePanelEnvironmentSwitches = ({
                             <StyledSubLabel>
                                 {strategiesLabel}
                                 {variantsLink}
+                                <ConditionallyRender
+                                    condition={
+                                        variants.length == 0 &&
+                                        environment.enabled &&
+                                        someEnabledEnvironmentHasVariants
+                                    }
+                                    show={
+                                        <HtmlTooltip
```

Comment:
We can extract this to const VariantsTooltip = <HtmlTooltip... so that show is a one liner

Reply:
I've extracted it to a component: https://github.com/Unleash/unleash/pull/2948/files#diff-b9629cb7366c55ca30629ef8fc0c813a999819557d3e886511c3e2001cd00db6 I was also concerned about having the content of the text duplicated.